lib/remote/jsonrpcconnection.cpp

```diff
@@ -209,9 +209,10 @@ void JsonRpcConnection::MessageHandler(const String& jsonString)
 		ApiFunction::Ptr afunc = ApiFunction::GetByName(method);
 
 		if (!afunc)
-			BOOST_THROW_EXCEPTION(std::invalid_argument("Function '" + method + "' does not exist."));
-
-		resultMessage->Set("result", afunc->Invoke(origin, message->Get("params")));
+			Log(LogWarning, "JsonRpcConnection")
+			    << "Call to non-existent function '" << method << "' from endpoint '" << m_Identity << "'.";
```

Comment:
Minor code style remark: I would put 2 lines in an if-condition into parenthesis. Not sure if compilers treat that well enough.